### PR TITLE
Fix cache & exclude CM samples

### PR DESCRIPTION
**Pull request by rogaldh**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,17 +42,17 @@ jobs:
...
       script:
         - |
-          ./scripts/gcloud.install.sh &&
-          ./scripts/k8s.install.sh &&
-          PATH_TO_FILE=${TRAVIS_BUILD_DIR}/k8s ./scripts/k8s.chart.download.sh &&
-          PATH_TO_CHARTS=${TRAVIS_BUILD_DIR}/k8s ./scripts/k8s.charts.apply.sh
+          scripts/gcloud.install.sh &&
+          scripts/k8s.install.sh &&
+          PATH_TO_FILE=${TRAVIS_BUILD_DIR}/k8s scripts/k8s.chart.download.sh &&
+          PATH_TO_CHARTS=${TRAVIS_BUILD_DIR}/k8s/[^_]** scripts/k8s.charts.apply.sh
```
Comment (manifest) [inline .travis.yml]: `[^_]**` ?

**Comment by ivanovaleksey** [comment]
@rogaldh there is a type in a _samples_ word

File: .travis.yml (hunk 1 above)
Comment (rogaldh) [reply .travis.yml]: There is prefixed configmap sample at k8s directory. That regexp allows to use unprefixed charts only.

Comment (manifest) [reply .travis.yml]: If your're talking about `signals-config.yaml` it should be rewritten by configmap from k8s repository.

Comment (manifest) [reply .travis.yml]: I see, there is a misspelling in the file extension. We use '.yml' here and '.yaml' everywhere else. Lets use `.yaml`.

Comment (rogaldh) [reply .travis.yml]: I'm talking about `_signals-config.yml`. It seems a good practice to explicitly show that this configmap is just a sample.

**Pull request merged**